cpp/example-control.cc

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+
+#include "zendar-api.h"
+#include "reqrep.pb.h"
+
+int main(int argc, char* argv[]) {
+  if (argc < 2) {
+    printf("Usage: example-control URI");
+  }
+  zendar::ZendarReceiver rcv(argv[1]);
+  zendar::ZendarError error;
+  zen_proto::control::Response rep;
+  rcv.Connect();
+  // Reset the ZPU
+  rcv.Stop();
```

Comment:
I believe we talked about moving this under the `rep.status().state()` so the `Stop()` is only executed when the ZPU is running.

Reply:
I think it's fine here, since you can `Stop()` a ZPU in any state, and we want to indicate that. In particular, there are states like `FAILED` that you want to stop.

Reply:
You know this, but the customer doesn't. I don't believe the various APIs are documented enough to tell the customer how each API operates and when it's OK to call the API vs. when it isn't. If you want to keep this code as-is, then you should have more detailed documentation on the behavior of the APIs.